app/app.go

```diff
@@ -716,6 +751,22 @@ func New(
 		authtypes.NewModuleAddress(adminmoduletypes.ModuleName).String(),
 	)
 
+	app.MarketMapKeeper = marketmapkeeper.NewKeeper(
+		runtime.NewKVStoreService(keys[marketmaptypes.StoreKey]),
+		appCodec,
+		authtypes.NewModuleAddress(adminmoduletypes.ModuleName),
+	)
+	marketmapModule := marketmap.NewAppModule(appCodec, app.MarketMapKeeper)
+
+	oracleKeeper := oraclekeeper.NewKeeper(runtime.NewKVStoreService(keys[oracletypes.StoreKey]),
+		appCodec,
+		app.MarketMapKeeper,
+		authtypes.NewModuleAddress(adminmoduletypes.ModuleName))
+	app.OracleKeeper = &oracleKeeper
+	oracleModule := oracle.NewAppModule(appCodec, *app.OracleKeeper)
```

Comment:
1) `NewKeeper` returns **a structure** on L761;
2) `app.OracleKeeper` is a pointer to a structure;
3) `NewAppModule` requires a structure, so you deref the pointer;

Maybe we should suggest Skip to use pointers everywhere?

Reply:
lets do that, i felt that when wrote the code